lib/taskbook.js

```diff
@@ -325,8 +325,50 @@ class Taskbook {
     render.successCreate(task);
   }
 
+  _flattenDown(array, result) {
+    for (let i = 0; i < array.length; i++) {
+      const value = array[i];
+
+      if (Array.isArray(value)) {
+        this._flattenDown(value, result);
+      } else {
+        result.push(value);
+      }
+    }
+
+    return result;
+  }
+
+  _generateNumbers(value) {
+    let _start = value[0] || null;
+    let _end = value[1] || null;
+    const _arr = [];
+
+    if (!_start || !_end) {
+      return 0;
+    }
+
+    if (_start === _end) {
+      return _start;
+    }
+
+    if (!parseInt(_start, 10) < parseInt(_end, 10)) {
+      [_start, _end] = [_end, _start];
+    }
+
+    for (let i = parseInt(_start, 10); i <= parseInt(_end, 10); i++) {
```

Comment:
You use parseInt on both start and end twice, why not assign this one time and reuse?